CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 - **General**: Internal gRPC connection's certificates are hot reloaded ([#6756](https://github.com/kedacore/keda/pull/6756))
 - **Metrics API**: Support multiple auth methods simultaneously in Metrics API scaler ([#6642](https://github.com/kedacore/keda/issues/6642))
 - **Temporal Scaler**: Support custom tlsServerName ([#6820](https://github.com/kedacore/keda/pull/6820))
+- **Datadog Scaler**: Fix bug with datadogNamespace config ([#6828](https://github.com/kedacore/keda/pull/6827))
```

Comment:
nit: the changelog validation script is unhappy
```
Validate Changelog..........................................................Failed
- hook id: validate-changelog
- exit code: 1

Checking section: Unreleased
Error: Wrong pattern found in section: Improvements , line: - **Datadog Scaler**: Fix bug with datadogNamespace config ([#6828](https://github.com/kedacore/keda/pull/6827))
```
I think it's just a PR number mismatch
```suggestion
- **Datadog Scaler**: Fix bug with datadogNamespace config ([#6828](https://github.com/kedacore/keda/pull/6828))
```

Reply:
Thanks Jan, I fixed it